common/openapi.yml

```diff
@@ -217,6 +218,8 @@ components:
           type: string
         dataObjectUri:
           type: string
+        ip:
+          type: string
```

Comment:
I think we actually dont want to add this here since IP will be in the header and not the request params

Reply:
maybe you can specify a required header in an openapi spec? I dont think i have before

Reply:
oh yeah duh, brain fart on my part